lib/src/services/constants.dart

```diff
@@ -1,3 +1,5 @@
+
+// todo : change to app router method in utility folder
```

Comment:
Yes, please move the constants to the app router

Reply:
done